specfile/tags.py

```diff
@@ -340,14 +341,29 @@ def __getitem__(self, i):
 
     def __delitem__(self, i: Union[SupportsIndex, slice]) -> None:
```

Comment:
Could you add at least a simple test case for the compression?

Reply:
It won't be so simple, but sure, I will :slightly_smiling_face: